src/viz-supervision-success/VizSupervisionSuccess.js

```diff
@@ -111,15 +111,15 @@ VizParoleSuccess.propTypes = {
         district: PropTypes.string.isRequired,
         projected_month: PropTypes.string.isRequired,
         projected_year: PropTypes.string.isRequired,
-        success_rate: PropTypes.string.isRequired,
+        success_rate: PropTypes.number.isRequired,
```

Comment:
The latest files provided had all `number` values for the `success_rate`.  Note that this matches the demographic `success_rate` file and expected type above.